applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_3d.py

```diff
@@ -1,22 +1,7 @@
 import KratosMultiphysics
```

Comment:
This is no longer needed if we register the process. It would be great if we can get rid of the Python side. Maybe we can leave it for a future PR.